superset-frontend/lerna.json

```diff
@@ -19,4 +23,4 @@
     }
   },
   "$schema": "node_modules/lerna/schemas/lerna-schema.json"
-}
+}
```

Comment:
The newline is missing here

Reply:
Right... these are generated by lerna. They were getting caught by the pre-commit, but then it mods the files, which means you can't do the release... it's an annoying catch-22. I'll try to find a way around it in this or a subsequent PR. 

Actually, now that I look at CI, there's a similar thing where I need to prepend the ASF license to CHANGELOG.md files... so I'll have to add a script for that at the very least to get the PR through.